src/lib.rs

```diff
@@ -721,6 +721,9 @@ pub enum OfflineWalletSubCommand {
         /// Selects which policy should be used to satisfy the internal descriptor
         #[structopt(name = "INT_POLICY", long = "internal_policy")]
         internal_policy: Option<String>,
+        /// Allows 80-bytes of arbitrary data to a transacation output via OP_RETURN
+        #[structopt(name = "ADD_DATA", short = "add", long = "add_data")]
```

Comment:
I think for this one it's OK to not have a short flag since `-add` isn't very descriptive.
```suggestion
        #[structopt(name = "ADD_DATA", long = "add_data")]
```